library/Vanilla/Site/SiteSectionModel.php

```diff
@@ -49,6 +52,41 @@ public function addProvider(SiteSectionProviderInterface $provider) {
         }
     }
 
+    /**
+     * Register optional default route
+     *
+     * @param string $name
+     * @param array $route Array should contain Destination and Type.
+     *          eg: ['Destination' => 'discussions', 'Type' => 'Internal', 'ImageUrl' => 'layout.png']
+     */
+    public function addDefaultRoute(string $name, array $route) {
+        $this->defaultRoutes[$name] = $route;
+    }
+
+    /**
+     * Get default route options
+     *
+     * @return array
+     */
+    public function getDefaultRoutes(): array {
+        return $this->defaultRoutes;
+    }
+
+    /**
+     * Get layout options
+     *
+     * @return array
+     */
+    public function getLayoutOptions(): array {
+        $layouts = [
+            'discussions' => 'Discussions',
+            'categories' => 'Categories'
```

Comment:
I think these need to be translated.

Reply:
I think it is should be translated during rendering on back end or front end component, but not in core component.